controllers/ramenconfig_test.go

```diff
@@ -9,16 +9,33 @@ import (
 	. "github.com/onsi/gomega"
 	ramen "github.com/ramendr/ramen/api/v1alpha1"
 	"github.com/ramendr/ramen/controllers"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
 	"sigs.k8s.io/yaml"
 )
 
 func configMapUpdate() {
 	ramenConfigYaml, err := yaml.Marshal(ramenConfig)
 	Expect(err).To(Succeed())
 
-	configMap.Data[controllers.ConfigMapRamenConfigKeyName] = string(ramenConfigYaml)
+	retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		var err error
 
-	Expect(k8sClient.Update(context.TODO(), configMap)).To(Succeed())
+		err = k8sClient.Get(context.TODO(),
+			types.NamespacedName{Name: controllers.HubOperatorConfigMapName, Namespace: ramenNamespace},
+			configMap)
+		if err != nil {
+			return err
+		}
+
+		configMap.Data[controllers.ConfigMapRamenConfigKeyName] = string(ramenConfigYaml)
+
+		err = k8sClient.Update(context.TODO(), configMap)
+
+		return err
```

Comment:
Any reason to keep the return value of k8sClient.Update() in err? We can simplify this
to:

    return k8sClient.Update(context.TODO(), configMap)

Reply:
Done